src/target/riscv/riscv.c

```diff
@@ -3590,6 +3817,20 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.help = "Control dcsr.ebreaku. When off, U-mode ebreak instructions "
 			"don't trap to OpenOCD. Defaults to on."
 	},
+	{
+		.name = "etrigger",
+		.handler = riscv_etrigger,
+		.mode = COMMAND_ANY,
+		.usage = "set [vs] [vu] [nmi] [m] [s] [u] <mie bits>|clear",
+		.help = "Set or clear a single interrupt trigger."
+	},
+	{
+		.name = "itrigger",
+		.handler = riscv_itrigger,
+		.mode = COMMAND_ANY,
+		.usage = "set [vs] [vu] [nmi] [m] [s] [u] <mie bits>|clear",
```

Comment:
Detail: Here I'd add an underscore to make it very clear that `<mie_bits>` is a single argument:

```suggestion
		.usage = "set [vs] [vu] [nmi] [m] [s] [u] <mie_bits>|clear",
```

Reply:
Done.